components/layout-switcher.tsx

```
@@ -1,113 +1,240 @@
-              </span>
-              {isActive && (
-                <div className="w-2 h-2 bg-blue-500 dark:bg-blue-400 rounded-full"></div>
-              )}
+      {availableLayouts.map(({ key, name, description, color, icon, preview, isActive }) => {
+        const textClasses = getTextClasses(color, isActive);
+        return (
+          <button
+            key={key}
+            className={`w-full p-5 rounded-2xl transition-all duration-300 border-2 shadow-lg hover:shadow-xl transform hover:scale-[1.02] ${getColorClasses(color, isActive)}`}
+            onClick={() => changeLayout(key)}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)